docs/userguide/developer/documentation.md

```diff
@@ -110,6 +110,10 @@ To build the core documentation without executing examples use:
 
 ```bash
 make docs
+
+# The above command required sphinx-build command.
+# If it is not automatically found, try setting it explicitly.
+SPHINXBUILD="python -m sphinx.cmd.build" make docs
```

Comment:
Am I the only one who needed this fix?
Wondering if we can/should just set `SPHINXBUILD="python -m sphinx.cmd.build"` in the Makefile [here](https://github.com/NVIDIA/earth2studio/blob/main/docs/Makefile#L7)?

Reply:
I have never had to set that before (I never knew this could be a problem)